bukkit/src/main/java/net/william278/husktowns/BukkitHuskTowns.java

```diff
@@ -297,6 +304,12 @@ public Manager getManager() {
         return manager;
     }
 
+    @NotNull
+    @Override
+    public GUIManager getGUIManager() {
```

Comment:
Could we make "GUI" consistent between all class names? I.e "Gui" instead of "GUI"